strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/ArtifactController.java

```diff
@@ -105,7 +105,7 @@ public ResponseEntity upload(@ApiParam(value = "The storageId", required = true)
     @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
     @RequestMapping(value = "{storageId}/{repositoryId}/**",
                     method = RequestMethod.GET,
-                    consumes = MediaType.TEXT_PLAIN_VALUE)
+                    consumes = {MediaType.TEXT_PLAIN_VALUE, MediaType.APPLICATION_OCTET_STREAM_VALUE})
```

Comment:
This should be `produces = MediaType.APPLICATION_OCTET_STREAM_VALUE`, not `consumes`. This is a download method, not an upload -- you're not putting/posting, but getting.